extras/wait-for-bricks.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+#Copyright (c) 2018 Red Hat, Inc. <http://www.redhat.com>
+#This file is part of gluster-block.
+#
+#This file is licensed to you under your choice of the GNU Lesser
+#General Public License, version 3 or any later version (LGPLv3 or
+#later), or the GNU General Public License, version 2 (GPLv2), in all
+#cases as published by the Free Software Foundation.
+
+# This file waits for gluster block hosting volume bricks to be up. If all the
+# bricks are up, it exits with success status, otherwise waits for the $timeout
+# seconds before erroring out with the status about
+# online-bricks/expected-online-bricks
+
+LOGDIR="${GB_LOGDIR:-/var/log/gluster-block}"
+
```

Comment:
* Since this script will be run before one of gluster-blockd and tcmu-runner start, where the logdir might be absent, hence this script should ' # mkdir -p ${LOGDIR}' before it start anything.

* As we are anyway have to post a new revision, can you add one a line after line 22 

* may be you might have to miss to address my previous comment about trace output ?

* Also will it be possible to move the test script under '${srcdir}/tests/glusterfs/' ?

* last but not the least, please add tested-by/reviewed-by tags as part of the next revision.

Thanks @pranithk !